src/nxdoc/nuxeo-server/rest-api/allowed-hosts-configuration.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Allowed Hosts Configuration
+description: Allows filtering host headers with a whitelist.
+tree_item_index: 800
+---
+Nuxeo allows you to filter HTTP requests by their standard `host`, `x-forwarded-host` and specific [`nuxeo-virtual-host`]({{page space='nxdoc' page='configuration-parameters-index-nuxeoconf#nuxeovirtualhost'}}) headers values against a whitelist. Unknown values will lead the HTTP request to be rejected with an `SC_BAD_REQUEST` status code.
+
+{{#> callout type='info' heading='Multiple headers filtering'}}
+All _present_ headers are always filtered. A mix of allowed and forbidden hosts in the different headers will be rejected.
```

Comment:
"All _present_ headers are always filtered." isn't clear to me, what do you mean?

Reply:
If there is `Host` + `nuxeo-virtual-host`, and `Host` is valid, `nuxeo-virtual-host` will also be evaluated anyway but not `x-forwarded-host` as it is not present and wouldn't match anything in our set.

Reply:
Maybe add it explicitly.

Reply:
What about adding:
```
which means, you don't need to fill all headers, only the filled ones must be valid.
For example: <my upper explenation>
```

Reply:
<img width="767" alt="image" src="https://github.com/nuxeo/doc.nuxeo.com-content/assets/47983949/f21584dc-9e91-4dbe-aa30-2fbfb96e0ed7">